app/challengers/Knight.py

```diff
@@ -0,0 +1,45 @@
+class Hero:
+    def __init__(self, name: str,
+                 power: int,
+                 hp: int,
+                 armour: int = 0) -> None:
+
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+
+    @property
+    def hp(self) -> int:
+        return self._hp
```

Comment:
You do not have this attribute. Add it to init

Reply:
do it